lib/mayaUsd/ufe/UsdContextOps.h

```diff
@@ -26,6 +26,13 @@
 namespace MAYAUSD_NS_DEF {
 namespace ufe {
 
+struct SchemaTypeGroup
+{
+    std::string               name;
+    std::vector<pxr::TfToken> types;
```

Comment:
According to the coding standards struct/class members should be prefixed with "_"

Reply:
Also for consistency you can use "TfTokenVector" which is a typedef for std::vector<TfToken>